charts/pega/charts/pegasearch/values.yaml

```diff
@@ -1,6 +1,11 @@
 ---
 # Enter the number of search nodes for Kubernetes to deploy (minimum 1).
 replicas: 1
+# IMPORTANT: https://www.elastic.co/guide/en/elasticsearch/reference/current/important-settings.html#minimum_master_nodes
+# To prevent data loss, it is vital to configure the discovery.zen.minimum_master_nodes setting so that each master-eligible
+# node knows the minimum number of master-eligible nodes that must be visible in order to form a cluster.
+# This value should be configured using formula (n/2) + 1 where n is replica count or desired capacity
+minimumMasterNodes: 1
```

Comment:
IMPORTANT: https://www.elastic.co/guide/en/elasticsearch/reference/current/important-settings.html#minimum_master_nodes
# To prevent data loss, you must configure the discovery.zen.minimum_master_nodes setting so that each master-eligible
# node is set to the minimum number of master-eligible nodes that must be visible in order to form a cluster.
# Configure this value using the formula (n/2) + 1 where n is replica count or desired capacity.

Reply:
Updated